padle/padle.py

```diff
@@ -0,0 +1,801 @@
+                    if m is None or not m.on_na or m.name_en is None:
+                        return await send_cancellation_message(ctx, f"{id} was invalid. Please remove "
+                                                                    "this ID and try again.")
+                await self.config.monsters_list.set(data)
+                await ctx.tick()
+            except Exception as e:
+                await send_cancellation_message(ctx, "Something went wrong.")
+                await ctx.send(box(str(e)))
+        else:
+            return await send_cancellation_message(ctx, "Looks like no file was attached!")
+
+    @padleadmin.command()
+    async def list(self, ctx):
+        """Lists all possible PADles"""
+        dbcog = await self.get_dbcog()
+        monsters_list = await self.config.monsters_list()
+        monsters_list.sort()
+        result = []
+        for id in monsters_list:
+            m = dbcog.get_monster(int(id))
+            if m is None or not m.on_na or m.name_en is None:
+                # this should never happen but just in case
+                result.append(f"**Error**: {id} is a valid PADle but not a monster.")
+                continue
+            result.append(f"{MonsterHeader.menu_title(m, use_emoji=True).to_markdown()}")
+        for page in pagify("\n".join(result)):
+            await ctx.send(page)
+        await ctx.send(f"There are **{len(result)}** valid PADles!")
+
+    @padleadmin.command(aliases=["settmrw", "settmrwpadle"])
+    async def settomorrowpadle(self, ctx, id: int):
+        """Sets tomorrow's PADle to a specified monster ID"""
+        dbcog = await self.get_dbcog()
+        monsters_list = await self.config.monsters_list()
+        if id == 0:
+            confirmation = await get_user_confirmation(ctx, "Would you like to make tomorrow's PADle random?")
+            if confirmation is None:
+                return await send_cancellation_message(ctx, "Confirmation timeout")
+            if not confirmation:
+                return await ctx.send("No changes were made.")
+            await self.config.tmrw_padle.set(0)
+            return await send_confirmation_message(ctx, "Tomorrow's PADle will be random.")
+        if id not in monsters_list:
+            # for the most part, avoid abuse and ensure validness
+            return await ctx.send("This monster is not a valid PADle. If you would like to set it as tomorrow's"
+                                  " PADle, please add it to the list of PADles first.")
+        m = dbcog.get_monster(id)
+        title = MonsterHeader.menu_title(m, use_emoji=True).to_markdown()
+        confirmation = await get_user_confirmation(ctx, "Are you sure you would like to change tomorrow's PADle to "
+                                                        f"{title}?")
+        if confirmation is None:
+            return await send_cancellation_message(ctx, "Confirmation timeout")
+        if not confirmation:
```

Comment:
These can both run.  If it's deliberate that's fine, but I think you may have meant elif here.

Reply:
it shouldn't because of the return right?

Reply:
🤡